### DISCUSSION TITLE
Calibrations do not reset Z height

### DISCUSSION THREAD

**Discussion by erylflynn** [supermerill/SuperSlicer#2144] [Q&A]
So wanted to try and dial things in and was going to use the SuperSlicer calibrations.  Problem is it prints the first item then goes to the second but stays at the same height the top layer of the first.  I see the code issues M486 but my Marlin returns an unknown code to this.  What are my options to modify this even by hand?

**Comment by supermerill** [comment]
can I have your project file?
You should open an issue for that.
